.ci/setup_env_ubuntu.sh

```diff
@@ -102,6 +102,12 @@ main()
 		pkgs_to_install+=" ${rust_agent_pkgs[@]}"
 	fi
 
+	# The redis-server package fails to install if IPv6 is disabled. Let's
+	# check if that's the case and then enable it.
+	if [[ $(sudo sysctl -n net.ipv6.conf.all.disable_ipv6) -eq 1 ]]; then
```

Comment:
Nit: You don't need to use double square brackets here.

Reply:
@jodh-intel just updated it to remove the extra square bracket. Thanks!